engines/stark/gfx/openglactor.cpp

```diff
@@ -105,7 +105,7 @@ void OpenGLActorRenderer::render(const Math::Vector3d &position, float direction
 	const Common::Array<BoneNode *> &bones = _model->getBones();
 
 	if (!_gfx->computeLightsEnabled()) {
-		glColorMaterial(GL_FRONT_AND_BACK, GL_DIFFUSE);
+		// glColorMaterial(GL_FRONT_AND_BACK, GL_DIFFUSE);
```

Comment:
I think this is part of the WIP and will be reverted once the feature is there, right?